tasks/internal_reconcile_report_orphan/src/use_cases/adapter_interfaces/storage.py

```diff
@@ -10,5 +10,5 @@ class OrphansPageStorageInterface:
 
     def get_orphans_page(self,
                          orphan_record_filter: OrphanRecordFilter,
-                         logger: logging.Logger
+                         LOGGER: logging.Logger
```

Comment:
Here and elsewhere, switch to lower-case for loggers passed via parameter. Upper-case denotes class properties.

Reply:
changed LOGGER to lowercase for those passed via parameter